src/Data/Tree/AVL/Zipper.hs

```diff
@@ -173,16 +159,17 @@ withLocus action = do
 -- | Add current node identity to the set of nodes touched.
 mark :: Retrieves h k v m => String -> Zipped h k v m ()
 mark _msg = do
-    rev <- revisionHere
-    trail %= Set.insert rev
+    uses locus rootHash >>= \case
+      Just hash -> do
+        trail %= Set.insert hash
```

Comment:
How did it even build in the first place? :thinking: Don't we have `-Werror` here?